core/vm/evm.go

```diff
@@ -198,35 +197,6 @@ func (evm *EVM) Call(caller ContractRef, addr common.Address, input []byte, gas
 	if value.Sign() != 0 && !evm.Context.CanTransfer(evm.StateDB, caller.Address(), value) {
 		return nil, gas, 0, ErrInsufficientBalance
 	}
-	lockupContractAddress := LockupContractAddresses[[2]byte{evm.chainConfig.Location[0], evm.chainConfig.Location[1]}]
```

Comment:
You need to remove the lock in the function parameters

Reply:
done